pom.xml

```diff
@@ -70,7 +71,7 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.332.x</artifactId>
+                <artifactId>bom-2.401.x</artifactId>
                 <version>1466.v85a_616ea_b_87c</version>
```

Comment:
If you're changing the artifactId you'll also need to change the version to one that exists.  Your current build error is artifact not found.

Reply:
List of bill of materials releases by version are available from the [BOM releases page](https://github.com/jenkinsci/bom/releases) for active versions and in the ["Depending on older versions" table](https://github.com/jenkinsci/bom?tab=readme-ov-file#depending-on-older-versions) for inactive versions

Reply:
Thanks for pointing this mistake/